index.html

```diff
@@ -1,22 +1,59 @@
 <!DOCTYPE html>
-<html>
+<ht
+ml>
 <head>
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+
+<!-- JS, Popper.js, and jQuery -->
+<script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js" integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV" crossorigin="anonymous"></script>
+
 	<title>Attendance</title>
 	<link rel="stylesheet" href="style.css">
 </head>
 <body>
 	<b><font size=6>Attendance Calculator</font></b>
-	<a href="https://academia.srmist.edu.in/#View:My_Attendance" target="_blank">
-		<button>SRM Academia</button>
+	<a class="a" href="https://academia.srmist.edu.in/#View:My_Attendance" target="_blank">

```

Comment:
can we have a better class name here

Reply:
Implemented in eb31b00fe595c41cf526af9afff9453fb6aa9840